### PR TITLE
Diary Foods Table and Create New button

### PR DESCRIPTION
**Pull request by izaxveta**
#### Pivotal URL:
https://www.pivotaltracker.com/story/show/154615179
https://www.pivotaltracker.com/story/show/154615180

#### What does this PR do?
This PR adds a header to the foods table in the index view and a 'Create New' button below the header that, when clicked, navigates the user to the foods.html view where they can create a new food and add it to the foods chart. This PR also places the food table, 'Foods' header and 'Create New' button in a container to group all these items for later styling.

#### Where should the reviewer start?
Start in the index.html file to review how the header and button is rendered, and later in the diary.scss file to review the styling.

#### Any additional context you want to provide?
Currently, the 'Create New' button brings the user to the foods.html page - which works when running the server locally. This is what the ticket is asking for at the moment, but I am curious to know if that is what is expected to show in the url bar when clicking on the button in GitHub Pages. In GitHub Pages, do we expect to see 'index' in the url or 'index.html'? @corywest @ameseee 

#### Screenshots (if appropriate)
image

### CODE REVIEW

**Comment by marshallhouston** [comment]
I don't understand your question about index vs index.html for GitHub pages? We will either be on / or index.html, and per the card, once a user clicks on the create new button, they'll be taken to /foods.html like you have it.

Did I misunderstand your question?

File: lib/diary.scss [hunk 1]
```diff
@@ -82,3 +83,12 @@
 .calories-consumed {
   border-top: none;
 }
+
+.default-button {
```
Comment (marshallhouston) [inline lib/diary.scss:87]: 👍 on reusability in this styling of the button

**Review by marshallhouston — APPROVED** [review]

**Comment by izaxveta** [comment]
Apologies, I wasn't very clear. Currently, when a user clicks on 'Create New', they'll be navigated to the foods page and the url will look like this:
image

Because the code looks like this:
image

@corywest @ameseee I'm wondering if it should actually look like this: 
image

**Comment by marshallhouston** [comment]
The language on the card specifies “foods.html” so I’d defer to that.

**Pull request merged**